playground/mock/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CYANS UI PLAYGROUND</title>
+  <link href="https://fonts.googleapis.com/css?family=Karla" rel="stylesheet">
+  <link rel="stylesheet" href="https://cdn.rawgit.com/necolas/normalize.css/73b6b0c7/normalize.css">
+  <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+
+  <main class="Playground">
+    <aside class="PlaygroundSidebar">
+      <h1 class="PlaygroundTitle">CYANS UI PLAYGROUND</h1>
+
+      <div class="Menu">
+        <div class="Folding">
+          <span class="Folding__Heading">Welcome</span>
+        </div>
+
+        <div class="Folding">
+          <span class="Folding__Heading">Button</span>
+          <span class="Folding__Subitem" aria-selected>with text</span>
```

Comment:
possiamo creare tutti gli attributi `aria-*`che vogliamo, come detto noi li useremo solo come convenzione, non c'entra nulla con il discorso dell'accessibilità. `is-animating` però non sarebbe un attributo valido perchè, in teoria, dovrebbe essere `data-is-animating`. 

Potremmo comunque considerare di usare gli attributi `aria-*` per gli stati e tipo `is-animating` per gli stati "transitori".